homeassistant/components/tradfri/cover.py

```diff
@@ -26,9 +26,11 @@ async def async_setup_entry(
     api = tradfri_data[KEY_API]
     devices = tradfri_data[DEVICES]
 
-    covers = [dev for dev in devices if dev.has_blind_control]
-    if covers:
-        async_add_entities(TradfriCover(cover, api, gateway_id) for cover in covers)
+    entities = [
```

Comment:
Let's revert the platform changes.

Reply:
I do not agree in this. The old version contains 2 loops, first making an array with all relevant devices, then a loop over these devices.

The new version is a single loop, and thus more efficient.

Reply:
I can of course make these changes in a new PR, but it seems a bit overkill.

Reply:
I would have more comments if we want to change this. Let's save this for another PR.

Reply:
Removed….and thanks another PR brings me closer to the 300Commits “reward” 😀 and being among the 20th most active committers.